client/components/data/query-jetpack-partner-portal-stored-cards/index.ts

```diff
@@ -5,20 +5,28 @@ import { isFetchingStoredCards } from 'calypso/state/partner-portal/stored-cards
 import type { CalypsoDispatch } from 'calypso/state/types';
 import type { AppState } from 'calypso/types';
 
-const request = () => ( dispatch: CalypsoDispatch, getState: AppState ) => {
+interface Props {
+	paging: {
+		startingAfter: string;
+		endingBefore: string;
+	};
+}
+
+const request = ( paging: { startingAfter: string; endingBefore: string } ) => (
```

Comment:
I think you're just creating more work for yourself by using redux over react-query but whatever floats your boat 😆

Reply:
@atanas-dev , do you know if there is an official statement about whether we should be using React Query over Redux for anything new?

Reply:
I don't know about an explicit announcement but a lot of older redux libs have been migrated over to react-query over time. @tyxla maybe you can chime in with some up-to-date info? 🙂